tests/backup/backup_test.go

```diff
@@ -303,6 +309,232 @@ func TeardownForTestcase(contexts []*scheduler.Context, providers []string, Clou
 
 }
 
+var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
+	numberOfUsers := 30
+	numberOfGroups := 3
+	groupSize := 10
+	numberOfBackups := 9
+	users := make([]string, 0)
+	groups := make([]string, 0)
+	backupNames := make([]string, 0)
+	var contexts []*scheduler.Context
+	labelSelectors := make(map[string]string)
+	var backupLocationUID string
+	var customBucketName string
+	var cloudCredUID string
+	var CloudCredUidList []string
+	var appContexts []*scheduler.Context
+	var bkpNamespaces []string
+	var clusterUid string
+	var clusterStatus api.ClusterInfo_StatusInfo_Status
+	var customBackupLocationName string
+	var credName string
+	bkpNamespaces = make([]string, 0)
+	providers := getProviders()
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("Backup: Share large number of backups",
+			"Share large number of backups with multiple users and groups with View only, Restore and Full Access", nil, 82934)
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+	It("Share large number of backups", func() {
+		Step("Validate applications and get their labels", func() {
+			ValidateApplications(contexts)
+			log.Infof("Create list of pod selector for the apps deployed")
+		})
+
+		Step("Create Users", func() {
+			var wg sync.WaitGroup
```

Comment:
can you ad log.infoD here about this step so that from dashboard log we can know what step it is executing

Reply:
simaliarly in other places also

Reply:
Fixed it. Please refer [Aetos Dashboard](http://aetos.pwx.purestorage.com/resultSet/testSetID/74560)